### PR TITLE
feat(src): implement static formatting toolbar - I35

### PR DESCRIPTION
**Pull request by irmerk**
# Issue #35 

### Changes
- Adjust linting
- Install icons
  - Move into svg component
  - Include specifications
- Implement static toolbar
  - Utilize portal
  - Render custom icons
  - Adjust styling
  - Optimize event handler with pointer events
- Adjust plugin icon styling
  - Color change on active state
- Create grid styling
- Remove hover component
- Adjust schema
  - Implement underline case
  - Add code support for lists
- Modularize method functionality
- List functionality
  - Handle list blocks to and from

### Flags
- ~Researching `FormatToolbar` functionality~
- ~Needs all button functionality corrected~
- ~Needs color change on active (already bolded, for example)~
- Needs parameter functionality
- Needs fix for clicking list button on a list item twice.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -341,11 +345,13 @@ function MarkdownEditor(props) {
     const { children, mark, attributes } = props;
 
     switch (mark.type) {
+      case 'html':
```
Comment (DianaLease) [inline src/index.js]: why was html moved from matching the `code` case to matching the ### bold case?

File: src/index.js [hunk 2]
```diff
@@ -541,6 +547,23 @@ function MarkdownEditor(props) {
     event.preventDefault();
     return false;
   });
+  /**
+   * Render the static-editing toolbar.
+   */
+  const renderEditor = useCallback((props, editor, next) => {
+    const children = next();
```
Comment (DianaLease) [inline src/index.js]: what does rendering `next()` as the children do here? I don't fully understand this part.

File: src/index.js (hunk 1 above)
Comment (irmerk) [reply src/index.js]: This is a very good question.

Comment (jeromesimeon) [reply src/index.js]: I thought the answer was obvious, so that HTML would be bold?

Comment (DianaLease) [reply src/index.js]: but doesn't it make more sense to match the `code` style if it's html? I could be wrong though!

File: src/index.js (hunk 2 above)
Comment (irmerk) [reply src/index.js]: [This](https://docs.slatejs.org/guides/rendering#the-editor-itself) would be the best I can do on a source for this.

File: src/index.js (hunk 1 above)
Comment (irmerk) [reply src/index.js]: I reverted back to being with `code`

File: src/index.js (hunk 2 above)
Comment (DianaLease) [reply src/index.js]: Got it, thank you!

**Pull request merged**